requirements.txt

```diff
@@ -24,7 +24,7 @@ pyparsing>=2.4
 python-dateutil>=2.8.1
 pytz>=2021.3
 requests>=2.20
-ruamel.yaml>=0.16,<0.18  # See https://github.com/great-expectations/great_expectations/issues/4152
```

Comment:
Following this link, it shows that `context.test_yaml_config(yaml.dump(datasource_config))` is deprecated, and we still have a number of those calls. Should we get those fixed?

Reply:
I had assumed those were already removed in 1.0.

Reply:
Oh ya know what? If I filter out versioned docs, it shows we don't call it. You should be good to go with this 😄